frontend/apps/marketing/src/components/section/index.tsx

```diff
@@ -0,0 +1,28 @@
+import '@code-dot-org/component-library/cms/section/index.css';
+
+// Import the background image used in the pattern backgrounds on Contentful
+import bgPatternImage from '@public/images/bg-pattern-lines.png';
+
+// Export the Component Definition for use in Contentful Studio
+export {SectionContentfulComponentDefinition} from './sectionContentfulDefinition';
+import DSCOSection, {
+  SectionProps,
+} from '@code-dot-org/component-library/cms/section';
+
+const Section = (props: SectionProps) => {
```

Comment:
Can we move this into a separate `Section.tsx` file, please?(same as all other marketing components that are using some 'custom' logic)

I agree this file is reasonably short, but at the same time it's rather hard to read this file since many things are happening here (imports,exports,declarations, etc)